### PR TITLE
Write shoot skill

### PR DESCRIPTION
**Pull request by Yudai-8-om**
## Description
Shoot skill

tick returns the list of actions the robot can take.
is_done function returns True if Shoot takes place, otherwise, it returns False.

### CODE REVIEW

File: rj_gameplay/rj_gameplay/skill/shoot.py [hunk 1]
```diff
@@ -24,20 +24,25 @@ class Shoot(IShoot):
...
-        #Add more logic for aiming and kicking
         self.pivot_behavior = ActionBehavior('Pivot', self.pivot, self.robot) 
         self.kick_behavior = ActionBehavior('Kick', self.kick, self.robot)
         self.root.add_children([self.capture_behavior, self.pivot_behavior, self.kick_behavior])
         self.root.setup_with_descendants()
 
-    def tick(self, world_state: rc.WorldState, robot:rc.Robot) -> None:
-        self.root.tick_once(robot)
-        # TODO: change so this properly returns the actions intent messages
+    def tick(self robot:rc.Robot, ctx) -> None:
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/skill/shoot.py]:
```suggestion
    def tick(self, robot:rc.Robot, ctx) -> None:
```
Missing comma

File: rj_gameplay/rj_gameplay/skill/shoot.py [hunk 2]
```diff
@@ -24,20 +24,25 @@ class Shoot(IShoot):
...
+        self.root.tick_once(robot, ctx)
+     
+
+    #if Kick and Pivot is done, Shoot should be done.
+    def is_done(self, world_state: rc.WorldState) -> bool:
+    	if self.pivot.is_done(world_state) and self.kick.done():
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/skill/shoot.py]: @HussainGynai Is there a reason kick's method is just `done()` instead of `is_done()`?

**Review by kfu02 — APPROVED** [review]
Looks good, left one comment for you and one for Hussain.

File: rj_gameplay/rj_gameplay/skill/shoot.py (hunk 2 above)
Comment (HussainGynai) [reply rj_gameplay/rj_gameplay/skill/shoot.py]: Oh yeah, its changed in the passing pr, @Yudai-8-om yo should change this  to self.kick.is_done(worldstate)

**Review by HussainGynai — APPROVED** [review]
Just make the change for kick.is_done() and you'll be good!

**Pull request merged**